src/pages/CertificatesViewAll.js

```diff
@@ -1,69 +1,93 @@
 import React, { useContext } from 'react'
 import styled from 'styled-components'
-import { Grid } from '@digicatapult/ui-component-library'
+import { Grid, Spinner, Table } from '@digicatapult/ui-component-library'
 
 import Nav from './components/Nav'
 import Header from './components/Header'
 
 import { Context } from '../utils/Context'
+import { stateToStatus, NameCell } from './components/shared'
 import { personas } from '../App'
+import { useNavigate } from 'react-router-dom'
+import useAxios from '../hooks/use-axios'
+
+const headersMap = {
+  heidi: [
+    'Date',
+    'H2 Batch size',
+    'Electric energy use',
+    'Carbon Embodiment',
+    'Status',
+  ],
+  emma: ['Date', 'Commitment'],
+  reginald: [],
+}
 
 export default function CertificatesViewAll() {
   const { current } = useContext(Context)
+  const { data, loading, error, callApiFn: fetch } = useAxios()
+  const navigate = useNavigate()
+
   const persona = personas.find(({ id }) => id === current)
+  const url = `${persona.origin}/v1/certificate`
+
+  React.useEffect(() => {
+    if (!data) fetch({ url })
+  }, [data, fetch, url])
+
   return (
     <>
       <Nav />
-      <Header userFullName={persona.name} companyName={persona.company} />
-      <LeftWrapper area="timeline"></LeftWrapper>
-      <MainWrapper>
-        <Grid.Panel area="main">
-          <Container>
-            <Text>
-              Certificates View All: <br />
-              Certificate One <br />
-              Certificate Two <br />
-              ...
-            </Text>
-          </Container>
-        </Grid.Panel>
-        <Sidebar area="sidebar"></Sidebar>
-      </MainWrapper>
+      <Header
+        userFullName={persona.name}
+        companyName={error || persona.company}
+      />
+      <Timeline area="timeline"></Timeline>
+      <Main area="main">
+        {(loading || error) && (
+          <Spinner
+            text={error || 'loading...'}
+            size={'large'}
+            color={persona.background}
+          />
+        )}
+        {data && (
+          <Table
+            action={(item) =>
+              navigate(`/certificate/${item.original_token_id}`)
+            }
```

Comment:
Which element is clickable ( I am trying to figure out what do I need to click to trigger navigate )

Also what's gonna happen if the **`original_token_id`** does not exist ?